src/diffDOM/dom/apply.js

```diff
@@ -92,7 +92,7 @@ export function applyDiff(
             node.parentNode.replaceChild(
                 objToNode(
                     diff[options._const.newValue],
-                    node.namespaceURI === 'http://www.w3.org/2000/svg',
```

Comment:
objToNode is getting "insideSvg" param as true in any case if the node we are replacing is 'svg'